include/oneapi/dpl/pstl/hetero/dpcpp/sycl_traits.h

```diff
@@ -129,7 +129,7 @@ template <typename _ReduceValueType, typename _Compare>
 struct __pattern_min_element_reduce_fn;
 
 template <typename _Compare, typename _ReduceValueType>
-struct __pattern_minmax_element__reduce_fn;
+struct __pattern_minmax_element_reduce_fn;
```

Comment:
another component of the unrelated change.

Reply:
The same.

Reply:
Reverted.